shuup/front/settings.py

```diff
@@ -116,3 +116,9 @@
 #: }
 #:
 SHUUP_FRONT_OVERRIDE_SORTS_AND_FILTERS_LABELS_LOGIC = {}
+
+
+#: Maximum allowed file size for uploads when posting to
+#: `shuup.front.views.upload.media_upload`
+#:
+SHUUP_FRONT_MAX_UPLOAD_SIZE = 500000
```

Comment:
oh, this could be something around 10MB, 500MB is too much I guess 😛

Reply:
I think this is 0.5MB